scripts/examples/gn_silabs_example.sh

```diff
@@ -273,6 +273,8 @@ else
 
     # 917 exception. TODO find a more generic way
     if [ "$SILABS_BOARD" == "BRD4325B" ]; then
+        # TODO: remove this flag once ota is supported
+        optArgs+="chip_enable_ota_requestor=false "
```

Comment:
This is beyond the scope of this PR. Should be removed and put in another PR by itself

Reply:
Removed this ota requestor from script file, will add in another PR.